mbed_lstools/lstools_win7.py

```diff
@@ -198,3 +198,22 @@ def regbin2str(self, regbin):
         """! Decode registry binary to readable string
         """
         return filter(lambda ch: ch in string.printable, regbin)
+
+    def mount_point_ready(self, path):
+        """! Check if a mount point is ready for file operations
+        @return Returns True if the given path exists, False otherwise
+        @details Calling the Windows command `dir` instead of using the python
+        `os.path.exists`. The latter causes a Python error box to appear claiming
+        there is "No Disk" for some devices that are in the ejected state. Calling
+        `dir` prevents this since it uses the Windows API to determine if the
+        device is ready before accessing the file system.
+        """
+        stdout, stderr, retcode = self.run_cli_process('dir %s' % path)
```

Comment:
Does ```dir``` command behave same on all platforms? I guess it will have different behaviour on Windows and Linux.